### PR TITLE
Allows dionae to join younger then 17.

### PR DESCRIPTION
**Pull request by Yonnimer**
This should allow Dionae to make characters younger than 17 in the setup. Doesn't make much sense a treeman would have to meet an age requirement.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
I suggest that instead you give them the `NO_AGE_MINIMUM` spawn_flag

**Comment by Yonnimer** [comment]
It looks like they already have the spawnflag, although I might be wrong.

**Comment by mikomyazaki** [comment]
> It looks like they already have the spawnflag, although I might be wrong.

Yeah. Sorry, that actually doesn't do exactly what I thought it did. Your PR does the right thing for what you're wanting :+1:

**Comment by Yonnimer** [comment]
!review

File: html/changelogs/Yonnimer-dionaeage.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Allows Dionae to join younger then 17."
```
Comment (mikomyazaki) [inline html/changelogs/Yonnimer-dionaeage.yml]:
```suggestion
  - rscadd: "Allows Dionae to join younger than 17."
```

**Pull request merged**